src/main/java/uk/gov/hmcts/reform/juddata/scheduler/JrdLeafDataLoadStarter.java

```diff
@@ -45,9 +46,16 @@ public void postConstruct() throws Exception {
 
     @Scheduled(cron = "${scheduler.camel-leaf-router-config}")
     public void runJrdLeafScheduler() {
-        camelContext.getGlobalOptions().remove(IS_EXCEPTION_HANDLED);
-        camelContext.getGlobalOptions().remove(SCHEDULER_STATUS);
-        dataLoadUtil.setGlobalConstant(camelContext, LEAF_ROUTE);
-        producerTemplate.sendBody(startLeafRoute, "starting JRD leaf routes though scheduler");
+        try {
+            camelContext.getGlobalOptions().remove(IS_EXCEPTION_HANDLED);
+            camelContext.getGlobalOptions().remove(SCHEDULER_STATUS);
+            dataLoadUtil.setGlobalConstant(camelContext, LEAF_ROUTE);
+            producerTemplate.sendBody(startLeafRoute, "starting JRD leaf routes though scheduler");
+        } catch (Exception ex) {
+            log.error("::leaf-route failed::", ex.getMessage());
```

Comment:
Print the stack trace here instead of message

Reply:
https://stackoverflow.com/questions/10477607/avoid-printstacktrace-use-a-logger-call-instead

Reply:
I mean print ex object instead of ex.message()